editoast/editoast_schemas/src/train_schedule/train_schedule_options.rs

```diff
@@ -14,8 +14,17 @@ pub struct TrainScheduleOptions {
     #[derivative(Default(value = "true"))]
     #[serde(default = "default_use_electrical_profiles")]
     use_electrical_profiles: bool,
+
+    #[derivative(Default(value = "false"))]
```

Comment:
1. Default `bool` is already `false`. Check the doc [here](https://doc.rust-lang.org/std/primitive.bool.html#impl-Default-for-bool).
2. We should indicate to serde that the value can be omitted. 

```suggestion
    #[serde(default)]
```

Reply:
thanks for the explanation :-)